### PR TITLE
refactor: advanced version control

### PR DESCRIPTION
**Pull request by mirestrepo**
This PR dresses:

- [x] Move updating from Honeycomb remote out of quick start and into the Version Control Section
- [x] Move testing issues in Linux out of quick start and into Troubleshooting

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -3,18 +3,29 @@ id: troubleshooting
...
 
+## Inspecting Errors
 
 When developing electron apps there are two processes: `main`, and `renderer`.  In this case `main` corresponds to `electron-starter.js` and its console is wherever you called `npm run dev` or `electron .` from. `renderer` corresponds to the React App - this is everything else. The react app's console is in the electron/browser window and can be seen by using dev tools to inspect the window.  When running `npm run dev`, it should open by default.
 
-### Potential Issues
+In case you want to find out where the error is coming from when running the app locally:
+
+* Try reviewing logs in the terminal
+* Inspect element in your browser by openning your developer tools. For instance, in Chrome, this can be done via the menu View -> Developer or right-clicking and pressing inspect.
```
Comment (RobertGemmaJr) [inline docs/troubleshooting.md]: Are we using `-` for bullet points or `*`?

**Review by gtdang — APPROVED** [review]
Looks good to me.

**Review by RobertGemmaJr — APPROVED** [review]
Looks awesome! I'm not sure of any other way around users merging updates from honeycomb into their projects... is there any sort of notification method for letting users know that Honeycomb has a new merge to main? Out of the scope of this PR but I'm curious

File: docs/troubleshooting.md (hunk 1 above)
Comment (mirestrepo) [reply docs/troubleshooting.md]: I can go with `-`

Comment (mirestrepo) [reply docs/troubleshooting.md]: > Looks awesome! I'm not sure of any other way around users merging updates from honeycomb into their projects... is there any sort of notification method for letting users know that Honeycomb has a new merge to main? Out of the scope of this PR but I'm curious

I'm not sure of another way. Ideally we could turn this into a cli, and maybe a honey-comb-scripts package. We could look into `create-react-app` and borrow some inspiration!

**Pull request merged**